### DISCUSSION TITLE
loading lualine in a specific filetype

### DISCUSSION THREAD

**Discussion by bryant-the-coder** [nvim-lualine/lualine.nvim#510] [Q&A] [answered]
how can i load lualine on a specific filetype? like i dun wan it to load in css file. how can i achieve that??

**Comment by shadmansaleh** [comment, marked as answer]
Try [disabled filetypes](https://github.com/nvim-lualine/lualine.nvim#disabling-lualine) option.

**Comment by bryant-the-coder** [comment]
do u know how to load a plugin on a specific keymap?

EDIT: mayb a lil off topic

**Reply by shadmansaleh** [reply]
If you're using packer, you can use `keys` option in your plugin definition to load a plugin on certain keymap.

https://github.com/wbthomason/packer.nvim#specifying-plugins , https://github.com/wbthomason/packer.nvim#keybindings

**Reply by bryant-the-coder** [reply]
indont seem to understand that part

**Reply by shadmansaleh** [reply]
Which plugin do you want to load on which keymap ?

**Reply by bryant-the-coder** [reply]
hmm... toggleterm, nvimtree

**Reply by shadmansaleh** [reply]
You should have something like this
```lua
use {"akinsho/toggleterm.nvim", opt = true, keys = {{'n', [[C-\]]}}}
```
Packer will load toggleterm when you press `CTRL-\` in normal mode

**Reply by bryant-the-coder** [reply]
what does opt=true does? does it need to be added in every lazy loaded plugin?

**Reply by shadmansaleh** [reply]
No you don't. You don't need it in this case either. But I  put them on my config as markers :D

**Reply by bryant-the-coder** [reply]
can share ur nvim conf?

**Reply by shadmansaleh** [reply]
No, sorry.

**Reply by bryant-the-coder** [reply]
y :))

**Comment by bryant-the-coder** [comment]
i wanna ask againnnn... how can i change the icon in different mode, changing the theme according to the time
